src/chat/zulip.md

```diff
@@ -0,0 +1,180 @@
+# Zulip
+
+[Rust's Zulip](rust-lang.zulipchat.com) is used by a number of teams, notably the compiler and language
+teams, along with their working groups.
+
+Zulip can be an unintuitive platform to get started with. To get started, take a
+look at the [getting started
+guide](https://zulipchat.com/help/getting-started-with-zulip). For more detail,
+examine the [Zulip user documentation](https://zulipchat.com/help/)!
+
+## Where to go for help with using Zulip
+
+If you're testing a feature, or want to get help, the `#zulip` stream is the
+place to go. Like elsewhere, the best thing to do is to create a new topic with
```

Comment:
```suggestion
place to go. Like elsewhere, the best thing to do is to create a new topic
```

Reply:
maybe commit this suggestion, @Mark-Simulacrum ?